pkg/proxy/iptables/proxier.go

```diff
@@ -1108,6 +1108,18 @@ func (proxier *Proxier) syncProxyRules() {
 				// Currently we only create it for loadbalancers (#33586).
 				writeLine(natRules, append(args, "-j", string(svcXlbChain))...)
 			}
+
```

Comment:
I have an idea for future refactoring that will make this nice, so I made the smallest change I could.